src/v2/mappers/TeacherMapper.ts

```diff
@@ -26,6 +30,20 @@ export class TeacherMapper {
     return roles;
   }
 
+  getSortedTeacher ({ sort, order = OrderQAParam.ASC }: SortDTO): Sort | object {
+    if (!sort) sort = SortQATParam.LAST_NAME;
+
+    const orderBy = sort.split(',').map((field) => ({
```

Comment:
front-end will never send more than one value from enum, so splitting and mapping is reduntant, as well as trimming